feature/preview/src/main/res/values/strings.xml

```diff
@@ -29,6 +29,8 @@
     <string name="toast_video_capture_external_unsupported">External video capture not supported</string>
     <string name="stabilization_icon_description_preview_and_video">Preview is Stabilized</string>
     <string name="stabilization_icon_description_video_only">Only Video is Stabilized</string>
+    <string name="toast_hdr_photo_unsupported">HDR photo not supported</string>
+    <string name="toast_hdr_video_unsupported">HDR video not supported</string>
```

Comment:
These don't really give much information beyond what the toggle button UI already shows. Can we add more detailed information like:
* "UltraHDR photos not supported on this device" (when neither lens supports > 1 image capture mode for both single-stream and multi-stream)
* "UltraHDR photos not supported by current lens" (when both multi-stream and single-stream capture modes do not support > 1 image capture mode)
* "Multi-stream mode does not support UltraHDR photo capture for current lens" 
* "Single-stream mode does not support UltraHDR photo capture for current lens"
* "Video not supported while app is in image-only capture mode" (for this one you can just update the text in `toast_video_capture_external_unsupported`)
* "HDR video not supported on this device" (when neither lens supports HDR video)
* "HDR video not supported by current lens" (when only the current lens doesn't support HDR video)

All of this information should be available in the `SystemConstraints` and `CameraAppSettings` passed into `getCaptureToggleUiState`.